client/components/mma/switch/SwitchOptions.tsx

```diff
@@ -76,6 +76,14 @@ const buttonStuckCss = css`
 	}
 `;
 
+const fromAppHeadingCss = css`
+	${textSans.large({ fontWeight: 'bold' })}
+	color:${palette.brand[500]};
+	line-height: 1.35;
```

Comment:
The typography API has a `lineHeight` option you could use here and _looks_ like `regular` maps to 1.35 which is the value you're after:
https://guardian.github.io/csnx/?path=/docs/source-foundations_typography--headline#options

Reply:
Thanks!